src/App.tsx

```diff
@@ -53,7 +59,9 @@ function App(): JSX.Element {
           setRootState(classes.root);
           setToolbarState(classes.toolbar);
           setContentState(classes.content);
-          navigate('/reservation');
+          if (location.pathname === '/') {
```

Comment:
Could you explain the difference between adding this, and what it was before?

Reply:
Yeah, so this is a little fix to prevent always redirecting to the reservation page if you are signed in. So if you go directly to `/parking` or `/admin` once it determined you were logged in it would redirect you to /reservation.

We only want this redirect if the user tries to access `/`, which is currently a link to login, but if you are logged in it's pointless! :)

Reply:
Oh ok, thanks!